src/frontend/src/icp/utils/date.utils.ts

```diff
@@ -11,3 +11,5 @@ export const normalizeTimestampToSeconds = (timestamp: number | bigint): number
 			? ts / Number(MILLISECONDS_IN_SECOND)
 			: ts;
 };
+
+export const relativeTimeFormatter = new Intl.RelativeTimeFormat('en', { numeric: 'auto' });
```

Comment:
If not required, I would keep this formatter in `format.utils`. This way it remains scoped and we avoid one reference between two modules.